packages/nuxt/src/app/composables/fetch.ts

```diff
@@ -144,7 +144,7 @@ export function useFetch<
     watch: watch === false ? [] : [_fetchOptions, _request, ...(watch || [])],
   }
 
-  if (import.meta.dev && import.meta.client) {
+  if (import.meta.dev) {
```

Comment:
The warning is only triggered on the server so in fact the check should be:

```suggestion
  if (import.meta.dev && import.meta.server) {
```